Extensions/Manage/Dnn.PersonaBar.Users/Components/RegisterController.cs

```diff
@@ -283,6 +283,9 @@ public UserBasicDto Register(RegisterationDetails registerationDetails)
                     case (int) Globals.PortalRegistrationType.VerifiedRegistration:
                         Mail.SendMail(newUser, MessageType.UserRegistrationVerified, portalSettings);
                         break;
+                    case (int)Globals.PortalRegistrationType.NoRegistration:
+                        Mail.SendMail(newUser, MessageType.UserRegistrationPublic, portalSettings);
```

Comment:
Did you verify, MessageType.UserRegistrationPublic contains the proper message text?

Reply:
I checked all the types and this is the type that best fits the scenario.

The registration email contains the following:

**Subject**
{{Website Name}} New User Registration

**Body**

Dear John Smith,

We are pleased to advise that you have been added as a Registered User to My Website. Please read the following information carefully and be sure to save this message in a safe location for future reference.

Portal Website Address: www.example.com
Username: john.smith@example.com

If you do not know, or cannot remember, your password, please go to http://example.com/default.aspx?ctl=PasswordReset&resetToken=95497741-47c7-4ebd-bcc3-462070526bab to reset it.
Thank you, we appreciate your support...

{{Website Name}}

Reply:
@Icidis thank you very much for the feedback!